songs/160_Jag_kan_lara_dig_c.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Jag kan lära dig C
+author: Arian och Sixten (DISK KM), DISK KM:s julfest 2011
```

Comment:
Given the apparent lack of an `event` tag I believe that this will suffice?

Reply:
Yes that looks good!